server/portal/apps/accounts/views.py

```diff
@@ -19,6 +21,28 @@
 logger = logging.getLogger(__name__)
 
 
+class LogoutView(DjangoLogoutView):
+    def dispatch(self, request, *args, **kwargs):
+        token = str(request.user.tapis_oauth.access_token)
+
+        logger.info('Attempting to revoke JWT token for user: ' + str(request.user))
+        self.revoke_token(token)
+
+        logout(request)
+
+        redirect = getattr(settings, 'LOGOUT_REDIRECT_URL', '/')
+        response = HttpResponseRedirect(redirect)
```

Comment:
My linter complained about redefining the `redirect` import from above
```suggestion
        redirect_uri = getattr(settings, 'LOGOUT_REDIRECT_URL', '/')
        response = HttpResponseRedirect(redirect_uri)
```